<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вакансии</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableVacancies">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSort">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Сортировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>250</y>
          <width>120</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="labelSearch">
        <property name="text">
         <string>Поиск по должности:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>